apps/indexers/nft-marketplace/schema.graphql

```diff
@@ -0,0 +1,58 @@
+type Account @entity {
+  id: ID!
+  tokens: [Token!]! @derivedFrom(field: "owner")
+}
+
+type NftContract @entity {
+  id: ID!
+  tokens: [Token!] @derivedFrom(field: "nftContract")
+}
+
+type Bid @entity {
+  auction: Auction!
+  price: BigInt!
+}
+
+type Auction @entity @index(fields: ["nftContract", "token"]){
+  token: Token! @index
+  nftContract: NftContract! @index
```

Comment:
Index on `nftContract` alone is redundant, because of `@index(fields: ["nftContract", "token"])`